societes/france.json

```diff
@@ -12378,6 +12457,38 @@
                 }
             ]
         },
+        {
+            "name": "Wiketio",
```

Comment:
Be careful, this is witekio and not wiketio.
It already exists (at the beginning of the json because it was Adeneo before). I know i need to do some cleanup to reorder all companies.

Reply:
fixed